src/components/Trackpad/ExtraKeys.tsx

```
@@ -1,30 +1,40 @@
+  return (
+    <div className="bg-base-300 p-2 flex flex-col gap-2 shrink-0">
+      {ROWS.map((row, i) => (
+        <div key={i} className="flex gap-2 justify-center">
+          {row.map(k => (
+            <button
+              key={k}
+              className="btn btn-sm btn-neutral min-w-14
+"
+              onPointerDown={e => press(e, k)}
+            >
+              {k}
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)